presto-main/src/main/java/io/prestosql/operator/project/MergePages.java

```diff
@@ -172,17 +171,17 @@ private Page flush()
             return output;
         }
 
-        private static boolean containsNotLoadedLazyBlock(Page page)
+        private static boolean allBlocksLoaded(Page page)
```

Comment:
Can we move this logic to `Page` ? As other operators can use the same in the future

Reply:
This method is part of rather poor heuristics and not used anywhere else. Let's see it if becomes useful in other places.